packages/recom-server/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM python:3.11.4-slim-buster
+
+# set work directory
+WORKDIR /usr/src/app
+
+# set environment variables
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+
+RUN apt update
```

Comment:
Consider running apt-get update and apt-get install in a single RUN instruction to reduce the number of image layers and make the build more efficient:

RUN apt-get update && apt-get install -y python3-pip python3-dev libpq-dev postgresql-contrib

Reply:
I'll take note of that.